src/Water_Balance.f90

```diff
@@ -38,15 +38,6 @@ SUBROUTINE Water_Balance
 !
     Q_diff(nrc) = (Q_out(nrc) - Q_in(nrc) - Q_sum_trib)/ndelta(nrc)
 !
-! Only half the cell at the confluence contributes flow and we'll assume that the 
```

Comment:
For this subroutine - maybe at the top define what this subroutine is doing? Just so people who are new to the code know what it is calculating?